lib/ansible/modules/notification/rabbitmq_publish.py

```diff
@@ -88,13 +88,29 @@
       - A dictionary of headers to post with the message.
     default: {}
     type: dict
+  cafile:
+    description:
+      - CA file used during connection to the RabbitMQ server over SSL.
```

Comment:
I would also mention in the docs that the three options are required together. I.e. add here something like `If this option is specified, also I(certfile) and I(keyfile) must be specified.` and similar sentences for the other two options.

Reply:
Thanks again @felixfontein - I've added a comment like you suggested to each certificate related parameter description.